x-pack/plugin/mapper-unsigned-long/src/main/java/org/elasticsearch/xpack/unsignedlong/UnsignedLongDocValuesField.java

```diff
@@ -113,7 +113,7 @@ protected long toFormatted(int index) {
 
     /** Returns the 0th index value as an {@code long} if it exists, otherwise {@code defaultValue}. */
     public long get(long defaultValue) {
-        return getValue(0, defaultValue);
+        return get(0, defaultValue);
```

Comment:
What's going on here?

Reply:
If intentional, please correct the comment `otherwise {@code defaultValue}`

Reply:
Tiny consistency issue that I just lumped in when I was passing by.